paths_v2.schema.json

```diff
@@ -0,0 +1,117 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Relayer Paths",
+  "description": "Path metadata to relay packets between two different chains.",
+  "type": "object",
+  "required": [
+    "paths"
+  ],
+  "properties": {
+    "paths": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "required": [
+          "src",
+          "dst",
+          "channels"
+        ],
+        "properties": {
+          "src": {
+            "type": "object",
+            "required": [
+              "client-id",
+              "connection-id"
+            ],
+            "properties": {
+              "client-id": {
+                "type": "string",
+                "description": "#/$defs/client-id-description"
+              },
+              "connection-id": {
+                "type": "string",
+                "description": "#/$defs/connection-id-description"
+              }
+            }
+          },
+          "dst": {
+            "type": "object",
+            "required": [
+              "chain-name",
+              "client-id",
+              "connection-id"
+            ],
+            "properties": {
+              "chain-name": {
+                "type": "string"
+              },
+              "client-id": {
+                "type": "string",
+                "description": "#/$defs/client-id-description"
+              },
+              "connection-id": {
+                "type": "string",
+                "description": "#/$defs/connection-id-description"
+              }
+            }
+          },
+          "channels": {
+            "type": "array",
+            "items": [
+              {
+                "type": "object",
+                "required": [
+                  "src",
+                  "dst",
+                  "order",
+                  "version"
+                ],
+                "properties": {
+                  "src": {
+                    "$ref": "#/$defs/channel_info"
+                  },
+                  "dst": {
+                    "$ref": "#/$defs/channel_info"
+                  },
+                  "order": {
```

Comment:
Nit: you could call this field `ordering` and be consistent with the naming in the protocol spec.
<img width="408" alt="image" src="https://user-images.githubusercontent.com/6876468/159570431-ade40f3d-a3a5-4976-aa8b-14a4dacdf236.png">

Reply:
Consistency is good. I'm ok with this. 
Thanks for your set of eyes.